jpa/bin/pom.xml

```diff
@@ -45,8 +46,21 @@
         <db.delimiter>;</db.delimiter>
         <junit.jupiter.version>5.10.2</junit.jupiter.version>
         <javadb.lib>${glassfish.toplevel.dir}/javadb/lib</javadb.lib>
-        <sql.directory>${project.basedir}/persistence-tck/sql</sql.directory>
+        <sql.directory>../sql</sql.directory>
+        <persistence.api.version>3.2.0-M2</persistence.api.version>
     </properties>
+    <repositories>
+        <repository>
```

Comment:
consider having ee4j:parent parent (ideally there would probably be a TCK runner parent sitting between ee4j:parent and a runner for particular TCK; such runner can define common stuff, like configurations for sigtest, etc to avoid duplications) - that would bring in this repo (and few more goodies) as well as some level of consistency across various TCKs